agent/logging_test.go

```diff
@@ -130,30 +128,33 @@ func TestStartLogging(t *testing.T) {
 			ID:     "logging",
 			Secret: "task_secret",
 		},
-		taskConfig: &internal.TaskConfig{
-			Task: &task.Task{},
-		},
 	}
 
 	ctx := context.Background()
-	assert.NoError(agt.fetchProjectConfig(ctx, tc))
-	require.NotNil(t, tc.project)
-	assert.EqualValues(model.EvergreenLogSender, tc.project.Loggers.Agent[0].Type)
-	assert.EqualValues(model.SplunkLogSender, tc.project.Loggers.System[0].Type)
-	assert.EqualValues(model.FileLogSender, tc.project.Loggers.Task[0].Type)
+	task, project, expansion, _, err := agt.fetchProjectConfig(ctx, tc)
```

Comment:
`makeTaskConfig` below this already calls `fetchProjectConfig` when the project is not initialized in the `taskContext`. Do we still need to call this directly here?

Reply:
removed